charts/cht-user-management/values.yaml

```diff
@@ -1,19 +1,73 @@
 cht-user-management:
   replicaCount: 1
-
   service:
     port: 3000
-
   image:
     repository: public.ecr.aws/medic/cht-user-management
-    tag: ""  # Set this to the version of the docker image
+    tag: "1.4.1"  # Set this to the version of the docker image
```

Comment:
It would be great if we could update the [deploy steps to NOT](https://github.com/medic/cht-user-management/blob/main/scripts/deploy/medic-deploy.md#helm-commands) depend on the `deploy` branch in CHT User Man repo:

> Switch to the deploy branch and ensure the same values file from the prior step is in the [values folder](https://github.com/medic/cht-user-management/blob/deploy/scripts/deploy/values/) in deploy branch

Maybe that's for another PR though?  Really - this is error prone and done manually for all prod pushes - a good recipe for a mistake to be made!

Reply:
So the deploy branch only contains the values yaml, and you want that moved to the main branch? Why is it in the deploy branch in the first place? Let's chat about it and put it in a different PR.

Reply:
https://github.com/medic/cht-user-management/pull/213

Here's the values file we want to use for users-chis-tg deploy.